pkg/sql/drop_test.go

```diff
@@ -916,6 +916,133 @@ CREATE TABLE t.kv (k CHAR PRIMARY KEY, v CHAR);
 
 }
 
+func TestDropTableUnsplitManuallySplitRanges(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+	params, _ := tests.CreateTestServerParams()
+
+	defer gcjob.SetSmallMaxGCIntervalForTest()()
+
+	s, sqlDB, kvDB := serverutils.StartServer(t, params)
+	defer s.Stopper().Stop(context.Background())
+	ctx := context.Background()
+
+	// Disable strict GC TTL enforcement because we're going to shove a zero-value
+	// TTL into the system with AddImmediateGCZoneConfig.
+	defer sqltestutils.DisableGCTTLStrictEnforcement(t, sqlDB)()
+
+	const numRows = 2*row.TableTruncateChunkSize + 1
+	const numKeys = 3 * numRows
+
+	tableName := "test1"
+	if err := tests.CreateKVTable(sqlDB, tableName, numRows); err != nil {
+		t.Fatal(err)
```

Comment:
This test will be much easier to read if you replace all these `if err := ...; err != nil { t.Fatal(err) }` blocks with `require.NoError(t, ...)`.